solvebio/query.py

```diff
@@ -852,6 +852,10 @@ def join(self, query_b, key, key_b=None, prefix="b_", always_prefix=False):
 
         # Prepare new returned query object
         new_query = self._clone()
+
+        # Set prefix in order to use it in a mutiple join
+        new_query._prefix = join_id + '_'
```

Comment:
whats the purpose of setting it to the _prefix attribute if it is only used in this function?